--- origsrc/quodlibet-2.3.1/quodlibet/formats/mod.py	2011-07-16 14:51:00.000000000 -0500
+++ src/quodlibet-2.3.1/quodlibet/formats/mod.py	2011-08-10 18:01:11.159389600 -0500
@@ -17,1 +17,1 @@ try:
-for so_version in ("libmodplug.so.1", "libmodplug.so.0"):
+for so_version in ("cygmodplug-1.dll", "cygmodplug-0.dll"):
--- origsrc/quodlibet-2.3.1/quodlibet/player/_xine.py	2011-07-16 14:51:00.000000000 -0500
+++ src/quodlibet-2.3.1/quodlibet/player/_xine.py	2011-08-10 18:01:21.599986800 -0500
@@ -12,1 +12,1 @@ import sys
-    _libxine = ctypes.cdll.LoadLibrary('libxine.so.2')
+    _libxine = ctypes.cdll.LoadLibrary('cygxine-2.dll')
@@ -16,1 +16,1 @@ import sys
-        _libxine = ctypes.cdll.LoadLibrary('libxine.so.1')
+        _libxine = ctypes.cdll.LoadLibrary('cygxine-1.dll')
