lxsession provides an incomplete org.gnome.GnomeSession API

--- origsrc/quodlibet-3.9.1/quodlibet/qltk/session.py	2017-05-26 06:05:06.000000000 -0500
+++ src/quodlibet-3.9.1/quodlibet/qltk/session.py	2017-06-13 17:50:46.877472700 -0500
@@ -41,7 +41,7 @@ def init(app_id):
 
         client_priv.connect_to_signal("QueryEndSession", query_end_session_cb)
         client_priv.connect_to_signal("EndSession", end_session_cb)
-    except dbus.DBusException:
+    except Exception:
         print_d("Connecting with the gnome session manager failed")
     else:
         print_d("Connected with gnome session manager: %s" % client_path)
