--- origsrc/quodlibet-1.0/Makefile	2007-04-15 21:45:47.000000000 -0500
+++ src/quodlibet-1.0/Makefile	2007-12-22 20:28:00.343750000 -0600
@@ -2,7 +2,7 @@
 MODULE_DIRS = browsers devices formats qltk parse plugins util library
 PROGRAMS = exfalso quodlibet
 SUBDIRS = trayicon mmkeys po
-EXTENSIONS = _trayicon.so _mmkeys.so
+EXTENSIONS = _trayicon.dll _mmkeys.dll
 PREFIX ?= /usr/local
 TO = share/quodlibet
 TODEP = lib/quodlibet
@@ -58,12 +58,12 @@
 
 extensions: $(EXTENSIONS)
 
-_trayicon.so:
-	cd trayicon && make trayicon.so && cd ..
-	cp trayicon/trayicon.so _trayicon.so
-_mmkeys.so:
-	cd mmkeys && make mmkeys.so && cd ..
-	cp mmkeys/mmkeys.so _mmkeys.so
+_trayicon.dll:
+	cd trayicon && make trayicon.dll && cd ..
+	cp trayicon/trayicon.dll _trayicon.dll
+_mmkeys.dll:
+	cd mmkeys && make mmkeys.dll && cd ..
+	cp mmkeys/mmkeys.dll _mmkeys.dll
 
 po-data:
 	cd po && make po
--- origsrc/quodlibet-1.0/formats/mod.py	2006-07-23 17:20:43.000000000 -0500
+++ src/quodlibet-1.0/formats/mod.py	2007-12-22 20:17:45.203125000 -0600
@@ -17,7 +17,7 @@
     '.mtm', '.okt', '.s3m', '.stm', '.ult', '.gdm', '.xm']
 try:
     import ctypes
-    _modplug = ctypes.cdll.LoadLibrary("libmodplug.so.0")
+    _modplug = ctypes.cdll.LoadLibrary("cygmodplug-0.dll")
 except (ImportError, OSError):
     extensions = []
 else:
--- origsrc/quodlibet-1.0/mmkeys/Makefile	2005-01-21 22:14:04.000000000 -0600
+++ src/quodlibet-1.0/mmkeys/Makefile	2007-12-22 20:28:26.171875000 -0600
@@ -1,7 +1,7 @@
 # Copyright 2004 Joe Wreschnig. Released under the terms of the GNU GPL.
 
-mmkeys.so: build
-	cp build/lib*/mmkeys.so .
+mmkeys.dll: build
+	cp build/lib*/mmkeys.dll .
 
 build: mmkeyspy.c
 	./setup.py build
@@ -17,7 +17,7 @@
 	mv gen-tmp $@
 
 clean:
-	rm -f mmkeys.so *.o mmkeyspy.c
+	rm -f mmkeys.dll *.o mmkeyspy.c
 	rm -rf build dist
 
 distclean: clean
--- origsrc/quodlibet-1.0/trayicon/Makefile	2005-11-25 22:14:20.000000000 -0600
+++ src/quodlibet-1.0/trayicon/Makefile	2007-12-22 20:28:41.203125000 -0600
@@ -1,7 +1,7 @@
 # Copyright 2004 Joe Wreschnig. Released under the terms of the GNU GPL.
 
-trayicon.so: build
-	cp build/lib*/trayicon.so .
+trayicon.dll: build
+	cp build/lib*/trayicon.dll .
 
 build: trayicon.c
 	./setup.py build
@@ -17,7 +17,7 @@
 	mv gen-tmp $@
 
 clean:
-	rm -f trayicon.so *.o trayicon.c
+	rm -f trayicon.dll *.o trayicon.c
 	rm -rf build dist
 
 distclean: clean
